### PR TITLE
Remove background color on slider

### PR DESCRIPTION
**Pull request by eskil0312**

### CODE REVIEW

File: src/components/general/Slider/styles.less [hunk 1]
```diff
@@ -45,7 +45,6 @@
         top: 50%;
         transform: translateY(-50%);
         z-index: 2;
-        background: var(--color-primary);
```
Comment (MartinForre) [inline src/components/general/Slider/styles.less]: Should probably apply similar changes to the markers aswell. Markers to the right of the dot are "highlighted"

Comment (eskil0312) [reply src/components/general/Slider/styles.less]: Remove style and the logic for applying it or just the style?

Comment (MartinForre) [reply src/components/general/Slider/styles.less]: Just style

Comment (eskil0312) [reply src/components/general/Slider/styles.less]: Removed style

**Pull request merged**